From 5e43bcc1b6311040615cd9cffaa2f18eb3e654d6 Mon Sep 17 00:00:00 2001
From: Paul Spooren <mail@aparcar.org>
Date: Sat, 4 Apr 2020 15:35:31 -1000
Subject: [PATCH 1/5] arm: dts: linksys: rename rango to wrt3200acm

Linksys uses internally the codename "rango" for a device sold under the
name of "Linksys WRT3200ACM", which is already reflected in the device
tree `model`. However the `compatible` list only contains the codename
`linksys,rango` which does not relate to the common name.

This patch renames the `dts` file to the sanitized model name and
prepends `linksys,wrt3200acm` to the `compatible` list to simplify the
device relation.

Signed-off-by: Paul Spooren <mail@aparcar.org>
CC: Rob Herring <robh+dt@kernel.org>
CC: Jason Cooper <jason@lakedaemon.net>
CC: Andrew Lunn <andrew@lunn.ch>
CC: Gregory Clement <gregory.clement@bootlin.com>
CC: Sebastian Hesselbarth <sebastian.hesselbarth@gmail.com>
CC: Daniel Golle <daniel@makrotopia.org>
CC: Adrian Schmutzler <freifunk@adrianschmutzler.de>
---
 arch/arm/boot/dts/Makefile                    |  2 +-
 ....dts => armada-385-linksys-wrt3200acm.dts} | 26 +++++++++----------
 2 files changed, 14 insertions(+), 14 deletions(-)
 rename arch/arm/boot/dts/{armada-385-linksys-rango.dts => armada-385-linksys-wrt3200acm.dts} (83%)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index e8dd99201397..a0df7f97cc44 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -1282,8 +1282,8 @@ dtb-$(CONFIG_MACH_ARMADA_38X) += \
 	armada-385-db-ap.dtb \
 	armada-385-linksys-caiman.dtb \
 	armada-385-linksys-cobra.dtb \
-	armada-385-linksys-rango.dtb \
 	armada-385-linksys-shelby.dtb \
+	armada-385-linksys-wrt3200acm.dtb \
 	armada-385-synology-ds116.dtb \
 	armada-385-turris-omnia.dtb \
 	armada-388-clearfog.dtb \
diff --git a/arch/arm/boot/dts/armada-385-linksys-rango.dts b/arch/arm/boot/dts/armada-385-linksys-wrt3200acm.dts
similarity index 83%
rename from arch/arm/boot/dts/armada-385-linksys-rango.dts
rename to arch/arm/boot/dts/armada-385-linksys-wrt3200acm.dts
index 3c4af57ec2b9..bf7b546e3344 100644
--- a/arch/arm/boot/dts/armada-385-linksys-rango.dts
+++ b/arch/arm/boot/dts/armada-385-linksys-wrt3200acm.dts
@@ -12,43 +12,43 @@
 
 / {
 	model = "Linksys WRT3200ACM";
-	compatible = "linksys,rango", "linksys,armada385", "marvell,armada385",
-		     "marvell,armada380";
+	compatible = "linksys,wrt3200acm", "linksys,rango", "linksys,armada385",
+		     "marvell,armada385", "marvell,armada380";
 };
 
 &expander0 {
 	wan_amber@0 {
-		label = "rango:amber:wan";
+		label = "wrt3200acm:amber:wan";
 		reg = <0x0>;
 	};
 
 	wan_white@1 {
-		label = "rango:white:wan";
+		label = "wrt3200acm:white:wan";
 		reg = <0x1>;
 	};
 
 	usb2@5 {
-		label = "rango:white:usb2";
+		label = "wrt3200acm:white:usb2";
 		reg = <0x5>;
 	};
 
 	usb3_1@6 {
-		label = "rango:white:usb3_1";
+		label = "wrt3200acm:white:usb3_1";
 		reg = <0x6>;
 	};
 
 	usb3_2@7 {
-		label = "rango:white:usb3_2";
+		label = "wrt3200acm:white:usb3_2";
 		reg = <0x7>;
 	};
 
 	wps_white@8 {
-		label = "rango:white:wps";
+		label = "wrt3200acm:white:wps";
 		reg = <0x8>;
 	};
 
 	wps_amber@9 {
-		label = "rango:amber:wps";
+		label = "wrt3200acm:amber:wps";
 		reg = <0x9>;
 	};
 };
@@ -56,22 +56,22 @@ wps_amber@9 {
 &gpio_leds {
 	power {
 		gpios = <&gpio1 24 GPIO_ACTIVE_HIGH>;
-		label = "rango:white:power";
+		label = "wrt3200acm:white:power";
 	};
 
 	sata {
 		gpios = <&gpio0 21 GPIO_ACTIVE_LOW>;
-		label = "rango:white:sata";
+		label = "wrt3200acm:white:sata";
 	};
 
 	wlan_2g {
 		gpios = <&gpio1 13 GPIO_ACTIVE_LOW>;
-		label = "rango:white:wlan_2g";
+		label = "wrt3200acm:white:wlan_2g";
 	};
 
 	wlan_5g {
 		gpios = <&gpio1 14 GPIO_ACTIVE_LOW>;
-		label = "rango:white:wlan_5g";
+		label = "wrt3200acm:white:wlan_5g";
 	};
 };
 
-- 
2.25.1

